pkg/util/util.go

```diff
@@ -239,3 +244,147 @@ func SplitQualifiedName(str string) (string, string) {
 func JoinQualifiedName(namespace, name string) string {
 	return path.Join(namespace, name)
 }
+
+type Route struct {
+	Interface   string
+	Destination net.IP
+	Gateway     net.IP
+	// TODO: add more fields here if needed
+}
+
+func getRoutes(input string) ([]Route, error) {
+	routes := []Route{}
+	inFile, err := os.Open(input)
+	if err != nil {
+		return nil, err
+	}
+	defer inFile.Close()
+	scanner := bufio.NewReader(inFile)
+	for {
+		line, err := scanner.ReadString('\n')
+		if err == io.EOF {
+			break
+		}
+		if strings.HasPrefix(line, "Iface") {
+			continue
+		}
+		fields := strings.Fields(line)
+		if len(fields) < 11 {
```

Comment:
I think this is correct as written.  It would weed out bogus lines.  A line with fewer than 11 fields would be an error and we should not proceed with it.